DevelopersGuide.adoc

```diff
@@ -616,7 +616,7 @@ Create a `deps.edn` file with this content:
            com.fulcrologic/fulcro {:mvn/version "3.0.10"}}
 
  :aliases {:dev {:extra-paths ["src/dev"]
-                 :extra-deps  {org.clojure/clojurescript   {:mvn/version "1.10.520"}
+                 :extra-deps  {org.clojure/clojurescript   {:mvn/version "1.10.1.10.742"}
```

Comment:
That diff cannot be right

Reply:
Ack! A little to hasty on the second attempt. Sorry about that.